csharp/ServiceAccountMaintenance/Program.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.ServiceAccount.V1Alpha1;
+
+namespace ServiceAccountMaintenance
+{
+    internal class Program
+    {
+        /// <summary>
+        ///     PLEASE REPLACE ME.
+        ///     Your SpatialOS project name.
+        ///     This scenario will create a new service account with read and write access to this project.
+        /// </summary>
+        private const string ProjectName = "platform_sdk_examples";
+
+        /// <summary>
+        ///     The threshold for when a service account's expiry time should be increased, in days relative to
+        ///     the current time. If the service account expires in fewer days than this, its lifetime will be extended
```

Comment:
You are correct. The numbers don't make too much sense but @lx223 wanted these values to be small so that it doesn't persist some service accounts for a long time. I'm going to alter them to something like 1 and 4 to make a bit more sense.